src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
@parjong I am sorry, I have not realized that the eeAllocUnwindInfo allocs just the tiny UnwindInfo with single entry.
Why don't we actually do the same thing as other platforms except for AMD64 and don't store the length in the FunctionLength directly in the RUNTIME_FUNCTION::UnwindData?